### PR TITLE
fix(huobi) - add tx 0x to eth network withdrawals

### PR DESCRIPTION
**Pull request by ttodua**
related #17027

I confirm that when withdrawing through ETH , the `0x` is not prefixed. idk, but we might need to report to huobi, as it could be extended to other networks, but we can't list them all here manually.

### CODE REVIEW

File: ts/src/huobi.ts [hunk 1]
```diff
@@ -5460,10 +5460,14 @@ export default class huobi extends Exchange {
             feeCost = Precise.stringAbs (feeCost);
         }
         const networkId = this.safeString (transaction, 'chain');
+        let txHash = this.safeString (transaction, 'tx-hash');
+        if (networkId === 'ETH' && !txHash.startsWith ('0x')) {
```
Comment (carlosmiei) [inline ts/src/huobi.ts:5464]: @ttodua couldn't we do a `indexOf('0x') == 0` check instead?

File: ts/src/huobi.ts [hunk 2]
```diff
@@ -5461,7 +5461,7 @@ export default class huobi extends Exchange {
         }
         const networkId = this.safeString (transaction, 'chain');
         let txHash = this.safeString (transaction, 'tx-hash');
-        if (networkId === 'ETH' && !txHash.startsWith ('0x')) {
```
Comment (carlosmiei) [inline ts/src/huobi.ts]: @ttodua isn't safer to do "indexOf('oX') == 0` ? to avoid matching this pattern in the middle of the string

Comment (ttodua) [reply ts/src/huobi.ts]: the thing is that there wouldn't be any `0x` occurrences inside transaction body, as they only include hexademicals  (0-9, a-f)

Comment (ttodua) [reply ts/src/huobi.ts]: ah, wait, i see the mistake, right.

Comment (ttodua) [reply ts/src/huobi.ts]: fixed

Comment (carlosmiei) [reply ts/src/huobi.ts]: @ttodua thanks

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**